Integrations/integration-Fortigate.yml

```diff
@@ -0,0 +1,1381 @@
+      type: string
+    - contextPath: Fortigate.Policy.ID
+      description: Policy ID
+      type: string
+    - contextPath: Fortigate.Policy.Description
+      description: Policy description
+      type: string
+    - contextPath: Fortigate.Policy.Status
+      description: The status of the poilcy (Enabled or Disabled)
+      type: string
+    - contextPath: Fortigate.Policy.Source
+      description: Source address
+      type: string
+    - contextPath: Fortigate.Policy.Destination
+      description: Destination address
+      type: string
+    - contextPath: Fortigate.Policy.Service
+      description: Service for the policy (for example HTTP)
+      type: string
+    - contextPath: Fortigate.Policy.Action
+      description: Policy action (Allow, Block)
+      type: string
+    - contextPath: Fortigate.Policy.Log
+      description: Does the policy log the traffic or not
+      type: string
+    - contextPath: Fortigate.Policy.Security
+      description: Policy attached security profile
+      type: string
+    description: Get information about a firewall policy on FortiGate
+  - name: fortigate-update-policy
+    arguments:
+    - name: policyID
+      required: true
+      description: Policy ID
+    - name: field
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - description
+      - status
+      - source
+      - destination
+      - service
+      - action
+      - log
+      - nat
+      description: Field parameter to update
+    - name: value
+      required: true
+      description: Value of field parameter to update
+    outputs:
+    - contextPath: Fortigate.Policy.Name
+      description: Policy name
+      type: string
+    - contextPath: Fortigate.Policy.ID
+      description: Policy ID
+      type: string
+    - contextPath: Fortigate.Policy.Description
+      description: Policy description
+      type: string
+    - contextPath: Fortigate.Policy.Status
+      description: The status of the poilcy (Enabled or Disabled)
+      type: string
+    - contextPath: Fortigate.Policy.Source
+      description: Source address
+      type: string
+    - contextPath: Fortigate.Policy.Destination
+      description: Destination address
+      type: string
+    - contextPath: Fortigate.Policy.Service
+      description: Service for the policy (for example HTTP)
+      type: string
+    - contextPath: Fortigate.Policy.Action
+      description: Policy action (Allow, Block)
+      type: string
+    - contextPath: Fortigate.Policy.Log
+      description: Does the policy log the traffic or not
+      type: boolean
+    description: Update a firewall policy on FortiGate
+    execution: true
+  - name: fortigate-create-policy
+    arguments:
+    - name: policyName
+      required: true
+      description: Policy name
+    - name: description
+      description: Description for the policy
+    - name: sourceIntf
+      required: true
+      description: 'Source interface  (example : port1/port2/port3)'
```

Comment:
The input should be `port1/port2/port3` or one of the three?

Reply:
one of them, will make it more clear